src/app/app.component.ts

```diff
@@ -26,4 +38,37 @@ export class AppComponent implements OnInit {
       }
     });
   }
+
+  private setMetasSubject(): void {
+    this.translations.onDefaultLangChange.subscribe((elem) => {
+    this.metasSubject.next(elem.translations.metas);
+  });
+}
+
+  private setTitleSubject(): void {
+     this.router.events
+    .pipe(
+     filter((events) => events instanceof NavigationEnd),
+     map((events) => (events as any).url.slice(1)),
+    )
+    .subscribe((nameTitle) => this.titleSubject.next(nameTitle));
+}
+  private installTitleAndMetaInBrowser(): void {
+   combineLatest(
+     this.titleSubject,
+     this.metasSubject,
+   )
+     .pipe(
+      filter(([title, metas]) => !!(title && metas)),
+      map(([title, metas]) => ({nameMeta: metas[title]})),
+      takeUntil(this.unsubscribe)
+     )
+     .subscribe(({nameMeta }) =>
+      this.titleAndMetaTagsService.initTitleAndMeta(nameMeta));
+ }
+
+  ngOnDestroy() {
```

Comment:
Please delete this hook, 'cause you need this subscription during app lifecycle

Reply:
fixed